BraveRewardsUI/Ads/AdsViewController.swift

```diff
@@ -334,11 +334,11 @@ extension AdsViewController {
     let notification = AdsNotification.customAd(
       title: Strings.myFirstAdTitle,
       body: Strings.myFirstAdBody,
-      url: URL(string: "https://brave.com/my-first-ad")!
+      url: "https://brave.com/my-first-ad"
     )
     
     adsViewController.display(ad: notification, handler: { (notification, action) in
-      completion(action, notification.targetURL)
+      completion(action, URL(string: notification.targetURL)!)
```

Comment:
can we do it safely and unwrap the `targetURL`?

Reply:
Fixed!